TodoListApp/TodoList.Api.Tests/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.2.0" targetFramework="net472" />
+  <package id="MSTest.TestFramework" version="1.2.0" targetFramework="net472" />
```

Comment:
It is not enough with a `NuGet` to remove a reference (project) you need to remove `NuGet` itself (from the NuGet manager or console)

Reply:
Preferably replace the `MSTest.TestAdapter` with `NUnit`'s test adapter, you would be able to run your `NUnit` tests even without R# this way…

Reply:
👍